sdk/config_helpers.go

```diff
@@ -856,18 +856,21 @@ func nginxPlusApiCallback(parent *crossplane.Directive, current *crossplane.Dire
 }
 
 func pingStubStatusApiEndpoint(statusAPI string) bool {
-	client := http.Client{Timeout: 50 * time.Millisecond}
+	client := http.Client{Timeout: 1 * time.Second}
```

Comment:
can we make this configurable? Saves us from having to PR it again in the future